jstests/oidc/lib/README.md

```diff
@@ -65,8 +65,8 @@ Currently, the following endpoints are implemented:
 - openid-configuration: `{issuer_url}/.well-known/openid-configuration` - for getting configuration,
 - token_endpoint: `{issuer_url}/token` - for creating an access token and refresh token (always equal to 'refresh_token'),
 - jwks_uri: `{issuer_url}/keys` - for getting the list of **JWKs**,
-- device_authorization_endpoint: `{issuer_url}/device/authorize` - to simulate _device authorization flow_,
-- token_introspection: `{issuer_url}/introspect` - to simulate _token introspection_.
+- device*authorization_endpoint: `{issuer_url}/device/authorize` - to simulate \_device authorization flow*,
+- token*introspection: `{issuer_url}/introspect` - to simulate \_token introspection*.
```

Comment:
This change seems to be unrelated to the FIPSMode handling and expressions in IDL files.
Looks like we can remove this change from this PR.
Also it is not clear what was the intention. Was it intended to italicize text between asterisks?

Reply:
This is in a separate commit and this is a result of running `bazel run format`. But sure I can move it into a separate PR.

Reply:
> This is in a separate commit and this is a result of running `bazel run format`. But sure I can move it into a separate PR.

For me this change looks like a nonsense:
- on the first line three words were surrounded by underscores, making them italic: device authorization flow. Now the first underscore is escaped and the last underscore is replaced with asterisk. That underscore should not be escaped because it represents markdown formatting.
- at the beginning of line there was an identifier `device_authorization_endpoint` which is broken after the change
- same issues on the second line

So it looks like `bazel run format` went crazy here.